src/main.cpp

```diff
@@ -809,7 +798,9 @@ int main(int argc, char *argv[]) {
 
   std::vector<std::string> found_kernels;
   dev_mgr.get_kernel_names(0, "ocl_Kernel", found_kernels);
-  cout << "Found Kernels: " << found_kernels.size() << endl;
+  if (benchmark_mode == false) {
+      cout << "Found Kernels: " << found_kernels.size() << endl;
+  }
```

Comment:
Why is this info not printed in benchmark mode?

Reply:
As one might want to log the console output during a benchmark run, it could lead to unexpected disk startups when the buffer is flushed to disk. This causes strange power spikes. As this output is primarily useful for debugging, I disabled the output in benchmark mode. Do you think it is necessary to output this information in benchmark mode?